src/domain/entities/schemas/page.schema.factory.ts

```diff
@@ -32,7 +32,6 @@ export function buildLenientPageSchema<T extends z.ZodTypeAny>(
 
     return {
       ...data,
-      count: results.length,
```

Comment:
As this is not used by the client (yet), I think it's fine to ignore the invalid counts but we should address it at some stage (reducing the count). Can you add a comment to reflect this?

Reply:
Addressed
https://github.com/safe-global/safe-client-gateway/pull/2195/commits/5f6e7ada9ce2871a7050aa741aeefc01c8910e4f